core/graph/src/main/java/uk/gov/gchq/gaffer/graph/Graph.java

```diff
@@ -285,7 +285,7 @@ public JobDetail executeJob(final Job job, final Context context) throws Operati
                 try {
                     result = graphHook.onFailure(result, clonedOpChain, clonedContext, e);
                 } catch (final Exception graphHookE) {
-                    LOGGER.warn("Error in graphHook " + graphHook.getClass().getSimpleName() + ": " + graphHookE.getMessage(), graphHookE);
+                    LOGGER.warn("Error in graphHook {}", graphHook.getClass().getSimpleName() ,": ", graphHookE.getMessage(), graphHookE);
```

Comment:
Confusingly, these usages are using the last argument, `graphHookE`, as a `Throwable`, and the first argument as a `String`. They look like [this](https://www.slf4j.org/api/org/slf4j/Logger.html#warn-java.lang.String-java.lang.Throwable-) rather than the other examples that look like [this](https://www.slf4j.org/api/org/slf4j/Logger.html#warn-java.lang.String-java.lang.Object...-).
A way to keep this working might be to change it like: 
```java
LOGGER.warn(String.format("Error in graphHook %s : %s", graphHook.getClass().getSimpleName(), graphHookE.getMessage()), graphHookE);
```